### DISCUSSION TITLE
How to compile gem with ffmpeg support?

### DISCUSSION THREAD

**Discussion by hellocatfood** [umlaeute/Gem#434] [Q&A] [answered]
I'm compiling Gem on Ubuntu 24.04 but it appears to not have ffmpeg support for loading videos. How can enable this?

**Comment by umlaeute** [comment, marked as answer]
you need all those packages:
- libavformat-dev
- libavutil-dev
- libavcodec-dev
- libswscale-dev
- pkgconf | pkg-config
